_inc/lib/tracks/tracks-ajax.js

```diff
@@ -1,7 +1,7 @@
 /* global jpTracksAJAX, jQuery */
-
 (function( $, jpTracksAJAX ) {
 	window.jpTracksAJAX = window.jpTracksAJAX || {};
+	const debugSet = localStorage.getItem( 'debug' ) === 'dops:analytics';
```

Comment:
Are we okay using `const` here? That file is loaded as is, it is not compiled later on.

Reply:
Yes it’s ok https://caniuse.com/#feat=const